docroot/modules/humsci/hs_layouts/hs_layouts.info.yml

```
@@ -6,6 +6,7 @@ version: 8.2.7
 package: 'Humanities & Sciences'
 dependencies:
   - layout_discovery
+  - drupal:dashboard
```

For contrib modules you should add the module name twice. `drupal:something` is only for core modules.

```suggestion
  - dashboard:dashboard
```